.github/workflows/lint-openapi.yaml

```diff
@@ -0,0 +1,20 @@
+name: "OpenAPI Lint"
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
+
+jobs:
+  openAPILint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      - name: OpenAPI Lint Checks
+        uses: nwestfall/openapi-action@v1.0.2
```

Comment:
Regarding this action, it seems this has been unmaintained for some time. And I checked alternatives and came across redocly-cli (MIT license) and used their docker image [1] to perform a similar validation 

You can see my attempt in the fork [2] and the test GH action failure run in my fork [3]

Let me know your thoughts on this :) 

[1] - https://github.com/Redocly/redocly-cli?tab=readme-ov-file#docker
[2] - https://github.com/open-feature/protocol/compare/test-validate
[3] - https://github.com/open-feature/protocol/actions/runs/8913421964/job/24478855581

Reply:
This is another option that adds lint violations as annotations in the PR.

https://github.com/stoplightio/spectral-action

Reply:
@beeme1mr checked but I am not super happy yet as there are open issues and PRs that were not addressed recently. We can switch back if we see issues with redocly